FirebaseRemoteConfigSwift/Tests/SwiftAPI/Codable.swift

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 import FirebaseRemoteConfig
-import FirebaseRemoteConfigSwift
```

Comment:
This should build whether or not, this line gets removed, right?

Reply:
Yep, correct.

Reply:
Manually verified. Resolving.